tgui/packages/tgui/interfaces/PreferencesMenu/jobs/jobs/head_of_security.ts

```diff
@@ -7,6 +7,7 @@ const HeadOfSecurity: Job = {
     make sure every department is protected, save the Virologist when he gets \
     mauled by monkeys.",
   department: Security,
+  alt_titles: ["Head of Security", "Security Commander"],
```

Comment:
```suggestion
  alt_titles: ["Head of Security", "Security Commander", "Sheriff],
```

I NEED funny cowboy title

also all the bri'sh ones...

Reply:
Were those already in? I don't remember seeing any of those in-game.

Reply:
they were, sheriff was used a lot

Reply:
Sheriff is being used by something else in the making.